### PR TITLE
feat: Add Locking DB and Redis configuration

### PR DESCRIPTION
**Pull request by Balsir**
This functionality exposes Locking DB and Redis configuration variables as chart values.

Setting the redis.password variable creates a secret for Redis password. 
Alternatively, set redisSecretName to use an existing Kubernetes Secret for the Redis password.

### CODE REVIEW

File: charts/atlantis/values.yaml [hunk 1]
```diff
@@ -413,3 +413,20 @@ servicemonitor:
...
+redis: {}
+#  host: redis.host.name
+#  password: myRedisPassword
+#  port: 6379
+#  db: 0
+#  tlsEnabled: false
+#  insecureSkipVerify: false
+
+# If managing secrets outside the chart for the Redis secret, use this variable to reference the secret name
+# redisSecretName: "myRedisSecret"
```
Comment (nitrocode) [inline charts/atlantis/values.yaml:431]: Why `redisSecretName` instead of putting `secretName` under `redis` ?

Comment (Balsir) [reply charts/atlantis/values.yaml:431]: `awsSecretName` and `apiSecretName` follow a similar convention.

**Review by nitrocode — DISMISSED** [review]

**Comment by Balsir** [comment]
Hi, thank you for approving the PR before. 
I took one more look and found a glaring typo which I fixed with https://github.com/runatlantis/helm-charts/pull/211/commits/3b5b5237b2549d78653be791cc84b1d30626e865. Sorry for the inconvenience.

**Review by nitrocode — APPROVED** [review]

**Comment by nitrocode** [comment]
Thank you @Balsir !

**Pull request merged**